c2rust-transpile/src/convert_type.rs

```diff
@@ -330,6 +330,7 @@ impl TypeConverter {
             CTypeKind::Int128 => Ok(mk().path_ty(mk().path(vec!["i128"]))),
             CTypeKind::UInt128 => Ok(mk().path_ty(mk().path(vec!["u128"]))),
             CTypeKind::BFloat16 => Ok(mk().path_ty(mk().path(vec!["bf16"]))),
+            CTypeKind::Float128 => Ok(mk().path_ty(mk().path(vec!["f128", "f128"]))),
```

Comment:
Can you combine this with the above `CTypeKind::LongDouble` arm?

Reply:
@kkysen I can. The main reason I hadn't was to allow for the theoretical possibility of handling targets where C's `long double` is not `f128`. However, it does seem unlikely that those would ever be supported, and if they *are* supported, we can always re-split the arms. Fixing.